src/app/home-page/home-page.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+
+import { CampaignService, SearchQuery } from '../campaign.service';
+import { CampaignSummary } from '../campaign-summary.model';
+
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.scss'],
+})
+export class HomePageComponent implements OnInit {
```

Comment:
I think this should be `HomeComponent` (and file names aligned) at least for now, for consistency with existing pages. If we think it's confusing we should rename everything en masse in a separate PR